src/test/java/org/jabref/logic/search/query/SearchQueryLuceneConversionTest.java

```diff
@@ -10,64 +12,55 @@
+                Arguments.of("content:\"two term\" annotations:\"two term\"", "\"two terms\""),
+                Arguments.of("content:\"two term\" annotations:\"two term\"", "any = \"two terms\""),
 
-                Arguments.of("( any:* AND -( groups:alpha ) ) AND ( any:* AND -( groups:beta ) )", "NOT(groups=alpha) AND NOT(groups=beta)"),
-                // not converted, because not working in JabRef 5.x
-                // Arguments.of("title:\"image processing\" OR keywords:\"image processing\"", "title|keywords = \"image processing\""),
+                Arguments.of("content:imag", "content = image"),
+                Arguments.of("annotations:imag", "annotations = image"),
+                Arguments.of("content:\"imag process\"", "content = \"image processing\""),
+                Arguments.of("+content:imag +annotations:process", "content = image AND annotations = processing"),
+                Arguments.of("+(content:imag annotations:process) +(content:term annotations:term)", "(content = image OR annotations = processing) AND term"),
+                Arguments.of("(content:on annotations:on) (+(content:two annotations:two) +(content:three annotations:three))", "one OR (two AND three)"),
 
-                // not converted, because wrong syntax for JabRef 5.x
-                // Arguments.of("( author:miller OR title:\"image processing\" OR keywords:\"image processing\" ) AND NOT author:brown AND NOT author:blue", "(author = miller or title|keywords = \"image processing\") and not author = brown and != author = blue"),
+                Arguments.of("(-content:term) (-annotations:term)", "any != term"),
+                Arguments.of("(-content:term) (-annotations:term)", "any !== term"),
+                Arguments.of("(-content:term) (-annotations:term)", "any !=! term"),
+                Arguments.of("(-content:\"two term\") (-annotations:\"two term\")", "any != \"two terms\""),
+                Arguments.of("+(-content:imag) +(-annotations:process)", "content != image AND annotations != processing"),
 
-                // String with a space
-                Arguments.of("title:image\\ processing AND author:smith", "title = \"image processing\" AND author= smith"),
+                Arguments.of("MatchNoDocsQuery(\"\")", "title = image"),
+                Arguments.of("content:\"imag process\" annotations:\"imag process\"", "\"image processing\" AND author = smith"),
+                Arguments.of("+(content:imag annotations:imag) +(content:process annotations:process)", "image AND (title = term OR processing)"),
+                Arguments.of("(content:imag annotations:imag) (content:process annotations:process)", "image OR (title = term OR processing)"),
+                Arguments.of("MatchNoDocsQuery(\"\")", "title = \"image processing\" AND author = smith"),
 
-                // We renamed fields
-                Arguments.of("any:somecontent", "anyfield = somecontent"),
```

Comment:
This should be kept? -- Not sure about the current state of discussion `any` versus `anyfield`.

Reply:
> This should be kept? -- Not sure about the current state of discussion `any` versus `anyfield`.

See https://github.com/JabRef/jabref/pull/11803#discussion_r1798130861